concent-builder/configure-jenkins.yml

```diff
@@ -0,0 +1,184 @@
+            owner: "jenkins"
+            group: "jenkins"
+          with_items:
+            - { path: "/users/admin/", name: "config.xml" }
+            - { path: "/users/admin/", name: "apiTokenStats.xml" }
+            - { path: "/", name: "credentials.xml" }
+            - { path: "/secrets/", name: "master.key" }
+            - { path: "/secrets/", name: "hudson.util.Secret" }
+
+        - name: Upload jenkins configuration file
+          # FIXME This need use as remote_src
+          copy:
+            src:   "jenkins-configuration-files/{{ item.name }}"
+            dest:  "/var/lib/jenkins{{ item.path }}{{ item.name }}"
+            owner: "jenkins"
+            group: "jenkins"
+          with_items:
+            - { path: "/jobs/Build-deploy-and-test/", name: "config.xml" }
+            - { path: "/", name: "github-plugin-configuration.xml" }
+            - { path: "/", name: "jenkins.model.JenkinsLocationConfiguration.xml" }
+
+        - name: Restart jenkins service
+          service:
+            enabled: yes
+            name:    jenkins
+            state:   restarted
+
+    - become:      yes
+      become_user: jenkins
+      block:
+        - name: Wait for jenkins after restart daemon
+          wait_for:
+            port:  8080
+            delay: 60
+
+        - name: Download jenkins-cli
+          get_url:
+            url:   "http://{{ jenkins_address }}/jnlpJars/jenkins-cli.jar"
+            dest:  "{{ jenkins_home_dir }}/jenkins-cli.jar"
+            mode:  0755
+
+        - name: Install needed plugin for jenkins
+          command: java                                               \
+              -jar jenkins-cli.jar                                    \
+              -s http://localhost:8080/                               \
+              -auth @/home/jenkins/concent-secrets/jenkins/.token     \
+              install-plugin                                          \
+              "{{ item }}"
+          args:
+            chdir: "{{ jenkins_home_dir }}/"
+          with_items:
+            - workflow-aggregator
+            - ansible
+
+        - name: Install github plugin for jenkins
+          command: java                                               \
+              -jar jenkins-cli.jar                                    \
+              -s http://localhost:8080/                               \
+              -auth @/home/jenkins/concent-secrets/jenkins/.token     \
+              install-plugin                                          \
+              github-oauth
+          args:
+            chdir: "{{ jenkins_home_dir }}/"
+
+    - become:      yes
+      become_user: root
+      block:
+        - name: Restart jenkins service
+          service:
+            enabled: yes
+            name:    jenkins
+            state:   restarted
+
+        - name: Add nginx config
+          copy:
+            src:   "jenkins-configuration-files/default"
+            dest:  "/etc/nginx/sites-enabled/default"
```

Comment:
I'd rather not put the configuration in `default`. Please call it `jenkins`.

You may also need to remove `default` but you should do this in the playbook that installs `nginx`. I.e. in `configure.yml`.